components/script/dom/htmlimageelement.rs

```diff
@@ -53,26 +58,40 @@ use servo_url::ServoUrl;
 use servo_url::origin::ImmutableOrigin;
 use std::cell::{Cell, RefMut};
 use std::char;
+use std::collections::HashSet;
 use std::default::Default;
 use std::i32;
 use std::sync::{Arc, Mutex};
-use style::attr::{AttrValue, LengthOrPercentageOrAuto, parse_double, parse_unsigned_integer};
+use style::attr::{AttrValue, LengthOrPercentageOrAuto, parse_double, parse_length, parse_unsigned_integer};
+use style::context::QuirksMode;
+use style::media_queries::{MediaQuery, MediaCondition};
+use style::parser::ParserContext;
 use style::str::is_ascii_digit;
+use style::stylesheets::{CssRuleType, Origin};
+use style::values::generics::transform::ToAbsoluteLength;
+use style::values::specified::{Length, source_size_list::SourceSizeList};
+use style_traits::ParsingMode;
 use task_source::{TaskSource, TaskSourceName};
 
+
 enum ParseState {
     InDescriptor,
     InParens,
     AfterDescriptor,
 }
 
-#[derive(Debug, PartialEq)]
+pub struct SourceSet {
+    pub imageSources: Vec<ImageSource>,
```

Comment:
let's use `snake_case` for the members, like: `image_sources` and `source_size`.